spec/helpers/application_helper_spec.rb

```diff
@@ -6,6 +6,15 @@
   include ApplicationHelper
   include LinkToHelper
 
+  describe '#can_ask_the_eu?' do
+
+    it 'delegates to WorldFOIWebsites.can_ask_the_eu?' do
+      expect(can_ask_the_eu?('US')).
+        to eq(WorldFOIWebsites.can_ask_the_eu?('US'))
```

Comment:
Actually, might just be better to check that the message and args are sent to this.